app/models/geographical_area.rb

```diff
@@ -80,4 +89,10 @@ def areas
   def id
     geographical_area_id
   end
+
+  private
+
+  def referenced_id
```

Comment:
We could do a migration to enable a reference but I'd rather hard code this in a hash for now as it very much could be yagni